packages/eslint-plugin/src/rules/array-type.ts

```diff
@@ -111,8 +111,10 @@ export default util.createRule<Options, MessageIds>({
           enum: ['array', 'generic', 'array-simple'],
         },
       },
-      prefixItems: [
+      items: [
```

Comment:
Hmm, looks the [`generated-rule-docs.ts` file](https://github.com/typescript-eslint/typescript-eslint/blob/72989344bf459552e3a4865c5bcbf44cf5d02a50/packages/website/plugins/generated-rule-docs.ts) will need to be updated to handle these new practices. Right now it assumes `meta.schema.prefixItems` exists on all schemas with the `'array'` type.

https://github.com/typescript-eslint/typescript-eslint/blob/72989344bf459552e3a4865c5bcbf44cf5d02a50/packages/website/plugins/generated-rule-docs.ts#L268

```plaintext
Compiled with problems:X

ERROR in ../eslint-plugin/docs/rules/array-type.md

Module build failed (from ../../node_modules/@docusaurus/mdx-loader/lib/index.js):
TypeError: Cannot read properties of undefined (reading '0')
    at /Users/josh/repos/typescript-eslint/packages/website/plugins/generated-rule-docs.ts:268:61
    at wrapped (/Users/josh/repos/typescript-eslint/node_modules/trough/wrap.js:25:19)
```

Reply:
Should be fixed now. Uses `items` instead of `prefixItems`, and also checks that it exists first.